solvers/julia_svrg.py

```diff
@@ -0,0 +1,44 @@
+from pathlib import Path
+from benchopt import safe_import_context
+
+from benchopt.helpers.julia import JuliaSolver
+from benchopt.helpers.julia import get_jl_interpreter
+from benchopt.helpers.julia import assert_julia_installed
+
+with safe_import_context() as import_ctx:
+    assert_julia_installed()
+
+
+# File containing the function to be called from julia
+JULIA_SOLVER_FILE = 'solvers/julia_stochopt.jl'
+
+
+class Solver(JuliaSolver):
+
+    name = 'julia-svrg'
+    references = [
+        'Gazagnadou, Gower, and Salmon,'
+        ' "Optimal mini-batch and step sizes for SAGA." '
+        'International conference on machine learning. PMLR, 2019.'
+        'https://github.com/gowerrobert/StochOpt.jl'
+    ]
+
+    # List of dependencies can be found on the package github
+    julia_requirements = [
+        'StochOpt::https://github.com/tommoral/StochOpt.jl#FIX_proper_module_install',  # noqa: E501
```

Comment:
Now that you merge tbng/StochOpt.jl#1, we can directly use it.
```suggestion
        'StochOpt::https://github.com/tbng/StochOpt.jl',
```